pandas/core/generic.py

```diff
@@ -1016,6 +1016,64 @@ def __setstate__(self, state):
     # ----------------------------------------------------------------------
     # I/O Methods
 
+    _shared_docs['to_excel'] = """
+    Write %(klass)s to a excel sheet
+
+    .. versionadded:: 0.20.0
```

Comment:
This is not the case for DataFrame. You can add it separately for series with `_shared_doc_kwargs`